src/test/test-udev.c

```diff
@@ -120,5 +118,5 @@ int main(int argc, char *argv[]) {
 out:
         mac_selinux_finish();
 
-        return r < 0 ? EXIT_FAILURE : EXIT_SUCCESS;
+        return EXIT_SUCCESS;
```

Comment:
hmm, i have the suspicion it would be better to return EXIT_FAILURE here in the two cases where the "out" label is used. In both cases this is done due to an error...